README.rst

```diff
@@ -13,20 +13,25 @@ Installation
 
 This code runs on Python 2.7.
 
-1.  Get a local copy of this repo.
+1.  Install standard development libraries. 
+    (ubuntu/debian):
 
-2.  (Optional)  Create and activate a virtualenv to work in.
+    $ sudo apt-get install python-dev libxml2-dev libxslt-dev lib32z1-dev  
```

Comment:
I put it before the virtualenv because we are installing the packages that virtualenv can't handle. Putting it after the creation of the virtualenv might be slightly misleading.